com.abagames.noiz2sa.json

```diff
@@ -0,0 +1,130 @@
+        },
+        {
+          "type": "patch",
+          "path": "patches/bulletml/bulletml_d.patch"
+        },
+        {
+          "type": "patch",
+          "path": "patches/bulletml/d_cpp.patch"
+        },
+        {
+          "type": "patch",
+          "path": "patches/bulletml/warnings.patch"
+        },
+        {
+          "type": "patch",
+          "path": "patches/bulletml/makefile.patch"
+        },
+        {
+          "type": "patch",
+          "path": "patches/bulletml/includes.patch"
+        },
+        {
+          "type": "patch",
+          "path": "patches/bulletml/get-rid-of-boost.patch"
+        }
+      ],
+      "build-commands": [
+        "make -C src CXXFLAGS=\"$(CXXFLAGS) $(CPPFLAGS) -fPIC -fpic\" LDFLAGS=\"$(LDFLAGS) -Wl,-z,defs\" libbulletml.so",
+        "mkdir -p /app/include/d/ /app/include/bulletml/ /app/include/bulletml/tinyxml/ /app/lib",
+        "cp src/bulletml.d /app/include/d/",
+        "cp src/*.h /app/include/bulletml/",
+        "cp src/tinyxml/*.h /app/include/bulletml/tinyxml/",
+        "cp src/lib*.so* /app/lib/"
+      ]
+    },
+    {
+      "name": "noiz2sa",
+      "no-autogen": true,
+      "no-make-install": true,
+      "subdir": "src",
+      "make-args": ["-f", "makefile.lin"],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://master.dl.sourceforge.net/project/noiz2sa/noiz2sa%20source_binary/Noiz2sa%200.51a/noiz2sa-0.51a.tar.gz",
+          "sha256": "a7e2b689a320b49cd87a5666b38b934528c65d5a212efca81a21fdd45c9f88e6"
+        },
+        {
+          "type": "patch",
+          "path": "patches/noiz2sa/0001-Use-shared-assets-location-from-a-config-header.patch"
+        },
+        {
+          "type": "patch",
+          "path": "patches/noiz2sa/0002-Remove-FoeCommand-class-namespace-from-method-in-cla.patch"
+        },
+        {
+          "type": "patch",
+          "path": "patches/noiz2sa/0003-Fix-makefile.patch"
+        },
+        {
+          "type": "file",
+          "path": "noiz2sa.desktop"
+        },
+        {
+          "type": "shell",
+          "commands": [
+            "install -Dm644 noiz2sa.desktop /app/share/applications/noiz2sa.desktop"
+          ]
+        },
+        {
+          "type": "file",
+          "path": "noiz2sa.png"
+        },
+        {
+          "type": "shell",
+          "commands": [
+            "install -Dm644 noiz2sa.png /app/share/icons/hicolor/64x64/apps/noiz2sa.png"
+          ]
+        },
+        {
+          "type": "file",
+          "path": "noiz2sa.appdata.xml"
+        },
+        {
+          "type": "shell",
+          "commands": [
+            "install -Dm644 noiz2sa.appdata.xml /app/share/appdata/noiz2sa.appdata.xml"
+          ]
+        }
+      ],
+      "build-commands": [
+        "install ./noiz2sa /app/bin/noiz2sa",
+        "install -d /app/share/games/noiz2sa",
+        "cp -R ../noiz2sa_share/* /app/share/games/noiz2sa"
```

Comment:
Can combine these:

```
install -m644 --target-directory /app/share/games/noiz2sa ../noiz2sa_share/*
```

Reply:
There are many manifest copying files in different ways. I don't think such changes should block PRs. I will update the manifest to do as you said, but let's not block things on nitpicks like these please, as any such improvements can come in later commits too.

Reply:
And in fact your suggestion seems to fail:
```
Running: install ./noiz2sa /app/bin/noiz2sa
Running: install -m644 --target-directory=/app/share/games/noiz2sa ../noiz2sa_share/*
install: failed to access '/app/share/games/noiz2sa': No such file or directory
```

So I will leave it as is.